README.md

```diff
@@ -103,15 +103,17 @@ with identical data and queried using identical queries.
 ## Installation
 
 TSBS is a collection of Go programs (with some auxiliary bash and Python
-scripts). The easiest way to get and install the Go programs is to use
-`go get` and then `make all` to install all binaries:
+scripts). The easiest way to get and install the Go programs is to clone
+the git repository and then `make all` to install all binaries:
+
 ```bash
-# Fetch TSBS and its dependencies
-$ go get github.com/timescale/tsbs
-$ cd $GOPATH/src/github.com/timescale/tsbs
+$ git clone https://github.com/timescale/tsbs.git
```

Comment:
I would suggest that the we should let the comment "# Fetch TSBS and its dependencies" stay.